maas-agent/src/helper.py

```diff
@@ -38,6 +38,20 @@ def uninstall() -> None:
         if maas.present:
             maas.ensure(SnapState.Absent)
 
+    @staticmethod
+    def refresh(channel: str) -> None:
+        """Refresh snap."""
+        maas = SnapCache()[MAAS_SNAP_NAME]
+        service = maas.services.get(MAAS_SERVICE, {})
+        maas.stop()
+        while service.get("activate", False):
+            pass
+        maas.ensure(SnapState.Present, channel=channel)
```

Comment:
we should use a *cohort* to guarantee we install the same revision in all nodes. (this is true for the `install()` method also)